oraclizeAPI_0.4.25.sol

```diff
@@ -31,21 +31,55 @@ THE SOFTWARE.
 
 // This api is currently targeted at 0.4.22 to 0.4.25 (stable builds), please import oraclizeAPI_pre0.4.sol or oraclizeAPI_0.4 where necessary
 
-pragma solidity >= 0.4.22 < 0.5;// Incompatible compiler version... please select one stated within pragma solidity or use different oraclizeAPI version
+pragma solidity >= 0.4.22 < 0.5; // Incompatible compiler version... please select one stated within pragma solidity or use different oraclizeAPI version
+
+contract OraclizeIBytes {
```

Comment:
comment explaining need for this segregation may be useful

Reply:
Kk, agreed. Comment added.